docs/resources/Channel.md

```diff
@@ -1111,10 +1111,11 @@ The `emoji` must be [URL Encoded](https://en.wikipedia.org/wiki/Percent-encoding
 
 ###### Query String Params
 
-| Field  | Type      | Description                           | Default |
-|--------|-----------|---------------------------------------|---------|
-| after? | snowflake | Get users after this user ID          | absent  |
-| limit? | integer   | Max number of users to return (1-100) | 25      |
+| Field  | Type      | Description                                      | Default |
+|--------|-----------|--------------------------------------------------|---------|
+| type?  | integer   | The reaction type (0 for normal and 1 for super) | 0       |
```

Comment:
Well cant we use there like reaction type ?

Reply:
The author of this PR can't change anything with the API and is only documenting what is already there, I believe your feedback is going to the wind

Reply:
well not that i meant
just make reaction type 

to show NORMAL = 0 and BURST = 1
easier to mention afterwards